zulip_bots/zulip_bots/bots/merels/libraries/game.py

```diff
@@ -0,0 +1,211 @@
+"""This is the main component of the game, the core of the game that squishes
+everything together and make the game work. Usually user can just import this
+module and use the beat() function and everything will be fine, but will there
+be any certain things that can't be accomplished that way, the user may also
+freely import another modules.
+"""
+
+import re
+
+from . import mechanics
+
+COMMAND_PATTERN = re.compile(
+    "^(\\w*).*(\\d,\\d).*(\\d,\\d)|^(\\w+).*(\\d,\\d)")
+
+
+def getInfo():
+    """ Gets the info on starting the game
+
+    :return: Info on how to start the game
+    """
+    return "To start a game, mention me and add `create`. A game will start " \
+           "in that topic. "
+
+
+def getHelp():
+    """ Gets the help message
+
+    :return: Help message
+    """
+
+    return "Commands: create, reset, put (v,h), move (v,h) -> (v,h), take (" \
```

Comment:
This could be a bit more detailed, e.g. add some explanations to the individual commands, a quick note on the game merels itself, etc.

Reply:
Added more info for the commands.